### ISSUE TITLE
libGDX Android on Nexus 6P 6.0.1 'has bad ELF magic'

### ISSUE THREAD

**Issue by AnnieTheEagle** [libgdx/libgdx#3785]
I was following the simple game tutorial, runs perfect on Desktop... Android, not so much. I get this on launch:

````
01-27 00:45:07.448  28202-28202/com.annietheeagle.dungeoneer E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: com.annietheeagle.dungeoneer, PID: 28202
    java.lang.ExceptionInInitializerError
            at java.lang.Class.newInstance(Native Method)
            at android.app.Instrumentation.newActivity(Instrumentation.java:1067)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2317)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
            at android.app.ActivityThread.-wrap11(ActivityThread.java)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:148)
            at android.app.ActivityThread.main(ActivityThread.java:5417)
            at java.lang.reflect.Method.invoke(Native Method)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load shared library 'gdx' for target: Linux, 32-bit
            at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:118)
            at com.badlogic.gdx.utils.GdxNativesLoader.load(GdxNativesLoader.java:34)
            at com.badlogic.gdx.backends.android.AndroidApplication.<clinit>(AndroidApplication.java:62)
            at java.lang.Class.newInstance(Native Method)
            at android.app.Instrumentation.newActivity(Instrumentation.java:1067)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2317)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
            at android.app.ActivityThread.-wrap11(ActivityThread.java)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:148)
            at android.app.ActivityThread.main(ActivityThread.java:5417)
            at java.lang.reflect.Method.invoke(Native Method)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: "/data/app/com.annietheeagle.dungeoneer-1/lib/arm64/libgdx.so" has bad ELF magic
            at java.lang.Runtime.loadLibrary(Runtime.java:372)
            at java.lang.System.loadLibrary(System.java:1076)
            at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:113)
            at com.badlogic.gdx.utils.GdxNativesLoader.load(GdxNativesLoader.java:34)
            at com.badlogic.gdx.backends.android.AndroidApplication.<clinit>(AndroidApplication.java:62)
            at java.lang.Class.newInstance(Native Method)
            at android.app.Instrumentation.newActivity(Instrumentation.java:1067)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2317)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
            at android.app.ActivityThread.-wrap11(ActivityThread.java)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:148)
            at android.app.ActivityThread.main(ActivityThread.java:5417)
            at java.lang.reflect.Method.invoke(Native Method)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
````

**Comment by AnnieTheEagle** [comment]
I get exactly the same error if I use a 64-bit x86_64 Emulator with HAXM.

**Comment by Tom-Ski** [comment]
What version of LibGDX?

**Comment by badlogic** [comment]
Must be latest release, i guess the copying you fixed tomski is the issue.

**Comment by AnnieTheEagle** [comment]
Hi, it was generated around two hours before the issue ticket was opened. The error is limited to 64-bit devices. I tried it on my brother's Nexus 5 and it works fine.

**Comment by badlogic** [comment]
Fixed via 1.9.1 release. Thanks for reporting.

**Comment by AnnieTheEagle** [comment]
Can confirm, 1.9.1 release after updating via build.gradle edit has fixed it and it's working fine now on my device.
